src/commandDetails/coin/leaderboard.ts

```diff
@@ -0,0 +1,72 @@
+import { container, SapphireClient } from '@sapphire/framework';
+import { Client, MessageEmbed } from 'discord.js';
+import {
+  CodeyCommandDetails,
+  CodeyCommandResponseType,
+  getUserIdFromMessage,
+  SapphireMessageExecuteType,
+  SapphireMessageResponse
+} from '../../codeyCommand';
+import { getCurrentCoinLeaderboard, getCoinBalanceByUserId, UserCoinEntry } from '../../components/coin';
+import { EMBED_COLOUR } from '../../utils/embeds';
+
+const getCurrentCoinLeaderboardEmbed = async (
+  client: SapphireClient<boolean>,
+  leaderboard: UserCoinEntry[],
+  currentUserId: string,
+  limit = 10
```

Comment:
maybe a const

Reply:
Do you think we could make this a parameter instead? Like in the command

Reply:
sure, probably cap at 150 tho, I don't want severe performance issues with ppl spamming .lb 2000

Reply:
I might cap it at 10 for now, since embeds have a max size. We can create an issue to introduce pagination to the leaderboards if needed to account for larger constants.

Reply:
For now the limit will be set to 10. But I'll keep the limit a parameter so that later on the leaderboard size can be increased.